### PR TITLE
Fixed ILRepack keeping file stream open after write

### PR DESCRIPTION
**Pull request by maritaria**
This is an issue when using the project as nuget lib

### CODE REVIEW

File: ILRepack/ILRepack.cs [hunk 1]
```diff
@@ -352,7 +352,9 @@ public void Repack()
                 Logger.Info("Output directory does not exist. Creating output directory: " + outputDir);
                 Directory.CreateDirectory(outputDir);
             }
-            TargetAssemblyDefinition.Write(Options.OutputFile, parameters);
+            using (FileStream outputStream = File.OpenWrite(Options.OutputFile))
+                TargetAssemblyDefinition.Write(outputStream, parameters);
```
Comment (timotei) [inline ILRepack/ILRepack.cs]: This looks like it should be fixed in upstream cecil, no?

Comment (lextm) [inline ILRepack/ILRepack.cs]: @timotei Cecil should have resolved that, http://cecil.pe/post/151940968861/disposable-ownership but 0.10 is still in beta.

Comment (gluck) [inline ILRepack/ILRepack.cs]: Current Cecil version in use seem to close the stream as well cf https://github.com/gluck/cecil/blob/c854cad1a1da59ea94e5e87cbcc0b0a1416c9b62/Mono.Cecil/ModuleDefinition.cs#L1080

Am I missing something ?

Comment (gluck) [inline ILRepack/ILRepack.cs]: There's a small difference (`FileAccess.ReadWrite` vs `FileAccess.Write`) but nothing that'd explain a stream kept open AFAICT.

Comment (timotei) [inline ILRepack/ILRepack.cs]: @gluck You linked `ModuleDefinition`, but this is on `AssemblyDefinition` which is different: https://github.com/gluck/cecil/blob/c854cad1a1da59ea94e5e87cbcc0b0a1416c9b62/Mono.Cecil/AssemblyDefinition.cs#L157

Comment (gluck) [inline ILRepack/ILRepack.cs]: It calls the module definition one no ?

Comment (timotei) [inline ILRepack/ILRepack.cs]: Hmm, no, it calls the AssemblyDefinition one AFAICT. There doesn't seem to be any relation between two (except that the assembly contains modules)

Comment (gluck) [inline ILRepack/ILRepack.cs]: I mean `AssemblyDefinition` one is only doing `main_module.Write (fileName, parameters);` and main_module is a ModuleDefinition, correct ?

Comment (timotei) [inline ILRepack/ILRepack.cs]: Oh, right, haven't dug deeper to check :(

**Comment by KirillOsenkov** [comment]
We have updated to the latest Cecil so this should no longer be an issue. Thanks!

**Pull request closed without merging**